index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/resources/css/style.css" />
+    <title>Portfolio White</title>
+  </head>
+  <body>
+    <!-- Nav bar -->
+    <main class="main__introduction background-primary">
+      <section class="nav">
+        <nav class="nav__container">
+          <div class="nav__logo">My Logo</div>
+          <div class="hamburguer-menu">
+            <img src="/resources/img/hamburger.png" alt="" />
+          </div>
+        </nav>
+      </section>
```

Comment:
- [ ] For semantic purposes, kindly replace the `<section>` tag here with a `<header>` tag, as this will help improve the SEO of your page. You can read more about the use of the header tag [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).